hoomd/md/methods.py

```diff
@@ -441,6 +443,20 @@ def thermalize_thermostat_and_barostat_dof(self):
         self._cpp_obj.thermalizeThermostatAndBarostatDOF(
             self._simulation.timestep)
 
+    @hoomd.logging.log
+    def thermostat_energy(self):
+        if not self._attached:
+            return None
+        else:
+            return self._cpp_obj.getThermostatEnergy(self._simulation.timestep)
```

Comment:
Is this a fix for #911?

Reply:
Well this only handles the NVT, NPT, and NPH methods, but I think so. The functions were necessary for C++ testing, so I went ahead and added the Python interface.

Reply:
Those are the methods that have a reservoir, so this PR will resolve #911.

Reply:
Is the Langevin/Berendsen energy tracking feature being removed/ do they use a different method?

Reply:
Neither Berendsen nor Langevin have equations of motion derived from a Hamiltonian, so there is no conserved quantity in those integrators to report.